src/karmabot/db/karma_note.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+import sqlalchemy as sa
+
+from karmabot.db.modelbase import SqlAlchemyBase
+
+
+class KarmaNote(SqlAlchemyBase):
+    """ Models a simple note system in the DB """
+
+    __tablename__ = "karma_note"
+
+    id: int = sa.Column(
+        sa.BigInteger().with_variant(sa.Integer, "sqlite"),
+        primary_key=True,
+        autoincrement=True,
+    )
+    user_id = sa.Column(sa.String, sa.ForeignKey("karma_user.user_id"), nullable=False)
+    timestamp = sa.Column(sa.DateTime, default=datetime.datetime.now, nullable=False)
+    note = sa.Column(sa.String)
+
+    def __repr__(self):

```

Comment:
this seems the `__str__` representation, no?

Reply:
I now implemented a proper str method, it was indeed more a repr method because the class name was present, now my str verison only prints the note, which is closer to what you would expect when printing the KarmaNote class or an instance

Reply:
LGTM